docs/pom.xml

```diff
@@ -87,6 +106,36 @@
                             </source>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>rename-iconset-files</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <source>
+                                def iconsets = new File(project.basedir, 'documentation/features/ui/iconset')
```

Comment:
In the Swagger PR Dancho moved that out to own groovy files for better maintenance. I think we should also do it here